### DISCUSSION TITLE
Fundamentos c# - Aula TextEditor

### DISCUSSION THREAD

**Discussion by syspone** [balta-io/forum#21] [Tira Dúvidas] [answered]
@andrebaltieri 

Quando salvo o aarquivo no caminho **C:\Users\cmsou\OneDrive\Documentos**  o mesmo retorna (Acesso Negado) 

![image](https://user-images.githubusercontent.com/48097255/191408348-c9be6abb-f436-4d1a-9a90-0f1e7d314f3c.png)

Ja autorizei via windows a pasta para acesso adm e liberei, mesmo assim nao consigo salvar.

Agora claro, o caminho c:/dev que o senhor passou na aula eu consigo salvar normalmente.

Esse é o caminho que estou usando;

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @cmsoouza, conforme mostra o erro, você não tem permissão de execução nesta pasta.
Recomendo seguir minhas dicas no curso e criar uma pasta `C:\dev` para executar os códigos.

O OneDrive tem permissões adicionais e não vai deixar você executar algo de lá.

**Reply by syspone** [reply]
blz professor, obg

**Reply by andrebaltieri** [reply]
💜
